probeinfo_api.yaml

```diff
@@ -481,7 +483,17 @@ components:
 
     Description:
       type: string
-      description: A brief free-text description of the repository
+      description: A brief free-text description of this application variant.
```

Comment:
Can we indicate this is optional now?

Reply:
I'm not sure if we want to do this. I'd encourage people to fill this field out, as it provides useful documentation that we can expose in the Glean Dictionary and possibly elsewhere.

Reply:
@fbertsch wdyt? ^

Reply:
Talked with Frank, just going to merge this. Can always followup later.